chirp/wxui/settingsedit.py

```diff
@@ -131,9 +131,11 @@ def _apply_settings(self):
                                 if setting[j].get_value() != val:
                                     # setting value modified or not accepted in
                                     # validate callback, propagate back to GUI
-                                    prop = self._propgrid.propgrid.GetProperty(
-                                        name)
-                                    prop.SetValue(setting[j].get_value())
+                                    prop = self._group_control \
+                                        .GetCurrentPage() \
+                                        .propgrid.GetProperty(name)
+                                    if prop:
+                                        prop.SetValue(setting[j].get_value())
```

Comment:
This is getting way too nested, and also using backslash continuations is not very conventional in chirp's style. However, I think using `page` per above will help.